src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -51,11 +51,11 @@
 (def edited-at-text (str " ⌫ " (i18n/label :t/edited)))
 
 (defn message-status [{:keys [outgoing content outgoing-status pinned edited-at in-popover?]}]
-  (when-not in-popover? ;; We keep track if showing this message in a list in pin-limit-popover
+  (when-not in-popover?                                     ;; We keep track if showing this message in a list in pin-limit-popover
```

Comment:
It is likely that those whitespace changes are not necessary

Reply:
yeah, i may need change the setting for my idea